samples/README.md

```diff
@@ -1,7 +1,8 @@
 ## Overview
 This folder contains some end-to-end samples using the pvsadm tool
 
-### 1. Image conversion and upload to pvs instance sample script
+### Script 1: Image conversion and upload to pvs instance sample script
+
 The image_create_and_upload_script helps to create ova image from qcow2 image, upload the images to a bucket (new/existing) and import them as boot image to pvs instance using the pvsadm tool
 #### Prerequisite
 1. Access to shell on IBM Power® logical partition (LPAR) running RHEL 8.x or CentOS 8.x with internet connectivity and minimum 250GB of free disk space
```

Comment:
Better to add - "You can spin up a CentOS instance in Power Virtual Server for image conversion"